### PR TITLE
Minor fixes and cleanup

### PR DESCRIPTION
**Pull request by OgelGames**

### CODE REVIEW

File: .luacheckrc [hunk 1]
```diff
@@ -1,22 +1,17 @@
-unused_args = false
-allow_defined_top = true
-
-exclude_files = {".luacheckrc"}
 
 globals = {
-    "minetest", "xcompat",
+     "minetest",
+	 "xcompat",
```
Comment (wsor4035) [inline .luacheckrc:4]: spacing

File: .luacheckrc [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
\ No newline at end of file
+    "default",
+	"mcl_sounds",
+	"ks_sounds",
+	"nodes_nature",
+	"fl_stone",
+	"fl_topsoil",
+	"fl_trees",
+	"hades_sounds",
+	"rp_sounds",
```
Comment (wsor4035) [inline .luacheckrc:16]: more spacing

File: README.md [hunk 3]
```diff
@@ -1,15 +1,30 @@
-# xcompat
+# XCompat [xcompat]
```
Comment (wsor4035) [inline README.md:1]: redundant

**Review by wsor4035 — CHANGES_REQUESTED** [review]
.

File: README.md (hunk 3 above)
Comment (OgelGames) [reply README.md:1]: Yeah, I guess it is... 🤔

**Review by BuckarooBanzay — APPROVED** [review]

**Pull request merged**